src/meteors/visualize/attr_visualize.py

```diff
@@ -148,7 +151,10 @@ def _merge_band_names_segments(band_names: dict[str | tuple[str, ...], int]) ->
 
 
 def visualize_spatial_attributes(
-    spatial_attributes: HSISpatialAttributes, ax: Axes | None = None, use_pyplot: bool = False
+    spatial_attributes: HSISpatialAttributes,
+    ax: Axes | None = None,
+    use_pyplot: bool = False,
+    title="Spatial Attributes Visualization",
```

Comment:
Sometimes you assign a default string and sometimes None and set the string in the code. I like when we assign a default string (less code) like here. Make it consistent please

Reply:
I will assign the default string everywhere. What about the first case, when we set the title with a method name?